ddtrace/contrib/internal/botocore/patch.py

```diff
@@ -229,4 +232,29 @@ def patched_api_call_fallback(original_func, instance, args, kwargs, function_va
             raise
         else:
             core.dispatch("botocore.patched_api_call.success", [ctx, result])
+            _prepare_span_pointers_from_successful_response(
+                ctx=ctx,
+                endpoint_name=endpoint_name,
+                operation_name=operation,
+                request_parameters=params,
+                response=result,
+            )
             return result
+
+
+def _prepare_span_pointers_from_successful_response(
+    ctx,
+    endpoint_name: str,
+    operation_name: str,
+    request_parameters: Dict[str, Any],
+    response: Dict[str, Any],
+) -> None:
+    span_pointer_descriptions = extract_span_pointers_from_successful_response(
+        endpoint_name=endpoint_name,
+        operation_name=operation_name,
+        request_parameters=request_parameters,
+        response=response,
+    )
+
+    for span_pointer_description in span_pointer_descriptions:
+        core.dispatch("datadog.span_pointer", [ctx, span_pointer_description])
```

Comment:
Multiple event dispatches seem unnecessary here. You could use a single `dispatch` call by passing the `span_pointer_descriptions` list.

Reply:
ah, will do!

Reply:
(well, this dispatch is completely gone for now)